script/hassfest/manifest.py

```diff
@@ -126,20 +126,23 @@ class QualityScale(IntEnum):
 ]
 
 
-def documentation_url(value: str) -> str:
+def own_documentation_url(value: str) -> str:
```

Comment:
Would `internal_documentation_url` or `native` or `ha` be better than `own`?

Reply:
Yea I renamed it to `core`